### ISSUE TITLE
Some weird behaviours

### ISSUE THREAD

**Issue by iago-pssjd** [rich-iannone/blastula#136]
First, thank you for the package. It can be very useful.

I open this issue by what seems to be a bug. I was following the example in the [RStudio blog](https://blog.rstudio.com/2019/12/05/emails-from-r-blastula-0-3/) and I got a weird behaviour (has it to do with issue #93?).
First, I try
```
library(blastula)
library(keyring)

create_smtp_creds_key(
  id = "my_id",
  user = "my@email.org",
  provider = "office365"
)

library(ggplot2)
library(glue)
plot <- qplot(disp, hp, data = mtcars, colour = mpg)
plot_email <- add_ggplot(plot)
email <- compose_email(
  body = md(c(
    "Team, How would you plot the relationship between these 3 variables?",
    plot_email
  ))
)

email %>%
  smtp_send(
    from = "my@email.org",
    to = "email@gmail.com",
    subject = "Testing the `smtp_send()` function",
    credentials = creds_key(id = "my_id")
  )
```
but I get
```
Error in curl::curl_fetch_memory(url, handle = h) : MAIL failed: 530
```
Then, following the suggestion [here](https://stackoverflow.com/a/59359269/997979), I add `use_ssl = TRUE` to `create_smtp_creds_key`, so:
```
create_smtp_creds_key(
  id = "my_id",
  user = "my@email.org",
  provider = "office365",
  use_ssl = TRUE
)
email %>%
  smtp_send(
    from = "my@email.org",
    to = "email@gmail.com",
    subject = "Testing the `smtp_send()` function",
    credentials = creds_key(id = "my_id")
  )
```
Then, the email is sent, but it is shown (both in the sending and in the reception addresses) as in the attached image:
![mail_ex](https://user-images.githubusercontent.com/40892925/71671611-c3f00f00-2d73-11ea-8452-b305873cd57b.png)
Thus, appearing html code, and with the `r` in the email body as an `=`.

Thank you!

**Comment by iago-pssjd** [comment]
Also with the previous example in the same link above, where
```
---
output: blastula::blastula_email 
--- 

Hi Team,

This *important* forecast needs to go out today.

```{r echo=FALSE}
model <- arima(presidents, c(1, 0, 0))
predict(model, 3)
```
is saved in the file `email.Rmd`, and then I send the email with (and again with `use_ssl = TRUE`) 
```
email <- render_email('P:/R/email.Rmd')

email %>%
  smtp_send(
    from = "my@email.org",
    to = "email@gmail.com",
    subject = "Testing the `smtp_send()` function",
    credentials = creds_key(id = "pssjd")
  )
```
I get a line `<td style="font-family: sans-serif; font-size: 14px; vertical-align: top;=>` in the shown body of the email.

I tested both examples with the CRAN and the development versions.

Thank you!

**Comment by rich-iannone** [comment]
Now fixed by #153
